img/i/id.me.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" version="1.1" id="Layer_1" x="0" y="0" xml:space="preserve" width="167.2" height="167.2"><style type="text/css" id="style27817">.st0{fill:#2d3e51}</style><g id="g28381" transform="translate(0 -53.8)"><path id="path27819" d="M8.4 107.4H5.5c-3.7 0-5.5 1.1-5.5 3.2v53c0 2.1 1.8 3.2 5.5 3.2h2.9c3.6 0 5.5-1.1 5.5-3.2v-53c0-2.1-1.9-3.2-5.5-3.2z" class="st0" fill="#2d3e51"/><path id="path27821" d="M59.7 160.4c0-6.3 4.1-11.7 9.8-13.5.7-2.9 1-6.2 1-9.7 0-19.3-9.7-29.8-27.2-29.8H24.6c-2.6 0-3.8 1.2-3.8 3.8v51.7c0 2.6 1.2 3.8 3.8 3.8h18.7c6.7 0 12.2-1.5 16.5-4.5-.1-.5-.1-1.2-.1-1.8zm-16.4-5.5h-8.6v-35.6h8.6c10.8 0 13 9.7 13 17.8 0 8.1-2.2 17.8-13 17.8z" class="st0" fill="#2d3e51"/><circle id="circle27823" r="6.5" cy="160.4" cx="73.9" fill="#2fc877"/><path id="path27825" d="M166.1 151.2c-.2-.1-.4-.1-.6-.1-.9 0-1.5.3-2 1.2-.3.5-.6 1-.9 1.6-1.2 2-2.4 4.1-4.2 5.6-2.4 2-5.9 3-8.9 2.4-1.8-.3-3-1.6-3.8-2.6-1.4-1.9-2-4.4-1.9-7.3 4.4-.4 17.6-2.3 18.7-12.3.2-1.9-.3-3.7-1.6-5.1-1.6-1.8-4.1-2.7-7.1-2.7-8.8 0-17.5 9.2-18.6 19.7-.3 2.9.1 5.5 1.1 7.9-.8.7-1.5 1.2-2.2 1.5-.7.4-1.4.4-1.8.2-.6-.3-.8-.9-.9-1.5-.3-1.9.1-4.1.6-6.4.3-1.3.7-2.7 1-4 1.1-3.8 2.2-7.7 1.8-11.7-.3-3.6-2.9-5.9-6.7-5.9-5.3 0-8.8 3.8-11 7 0-2.4-.6-4.2-1.7-5.3-1.1-1.1-2.6-1.7-4.7-1.7-5.2 0-8.6 3.6-10.9 6.8 0-.3.1-.6.1-.9.1-1.4.2-3.3-.9-4.5-.6-.7-1.6-1.1-2.9-1.1-1.1 0-1.4 0-2.6.2 0 0-1.2.2-1.6.6-.8.7-.5 1.7-.4 2.3v.2c.1.5.1 1.2.1 1.9-.2 3.8-.9 7.6-1.5 10.9-.4 1.7-.8 3.5-1.2 5.3-.9 3.9-1.8 7.9-2.3 11.9-.1.4.1.8.3 1.1.3.3.7.5 1.1.5h.3c2.9 0 5.7-.1 6.3-1.7.7-1.8 1.1-4 1.5-5.9l.2-.9c.8-4.4 1.6-7.6 3.1-11.7.8-2.1 2.2-4 3.3-5.4 1.4-1.7 2.9-3.4 4.5-3.4.7 0 1.1.2 1.5.6 1.6 2.2-.4 9.2-1.3 12.1-.2.6-.3 1.2-.4 1.6l-.6 2.3c-.9 3.4-1.8 6.8-2.2 10.4 0 .4-.1.7-.1 1.1v.5l.4.3c.7.6 6-.2 6-.2 1.9-.7 2.4-2.5 2.5-3.1.4-1.6.7-3.1 1-4.7v-.1c.6-2.9 1.2-5.9 2.1-8.7 1.7-5.6 4.1-9.4 6.9-11.4 1.2-.8 2.5-.9 3-.2 1 1.2.4 4.5.1 5.9-.4 1.9-.8 3.9-1.3 5.8v.1c-.3 1.2-.6 2.4-.9 3.7-.8 3.8-1.5 8.5.6 11.2 1.1 1.4 2.8 2.1 5 2.1 2.4 0 4.6-.8 7-2.5.7-.5 1.3-1 2.1-1.7 2.7 3.1 5.8 4.4 10.2 4.4 10 0 14.7-6.7 17.4-11.7.3-.6.7-1.4.9-2 .3-1.1-.1-2.1-1-2.5zm-12.3-12.6v1.2c-.1.8-.2 1.6-.5 2.3-1.4 4.6-5.5 5.5-8.9 5.6.5-2 1.2-4 2-5.7 1.7-3.5 3.8-5.6 5.6-5.6.2 0 .3 0 .5.1.1 0 .1.1.2.1h.2c.1 0 .1.1.2.2l.1.1.1.1c0 .1.1.1.1.2l.1.1c0 .1.1.1.1.2 0 0 0 .1.1.2 0 .1.1.2.1.4v.1c0 .1.1.3.1.5z" class="st0" fill="#2d3e51"/></g></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 96 96"><rect width="96" height="96" fill="#1d9f61" ry="18"/><path fill="#fff" d="M40 25v47h21c11 0 21-11 21-24S72 25 61 25Zm11 10h10c5 0 9 6 9 14s-4 14-9 14H51ZM19 25h11v47H19Z"/></svg>
```

Comment:
That icon only seems be used as the `favicon.ico`, but not anywhere else. However, these are notoriously out-of-date.

Reply:
Look how symertrical and small it is though :pleading_face:

Reply:
"That's what she said."